.github/workflows/node.yml

```diff
@@ -27,9 +27,6 @@ jobs:
         run: |
           go get -v -t -d ./...
 
-      - name: Build
-        run: go build -v ./...
```

Comment:
Does this reuse the preexisting binary that is cached?

Reply:
No, yarn chain:build ends up do a `make build`, so this is unnecessary and broken.

Reply:
Ahh that's right it does too.